kuksa-client/kuksa_client/__init__.py

```diff
@@ -51,32 +51,32 @@ def checkConnection(self):
     def stop(self):
         self.backend.stop()
 
-    # Do authorization by passing a jwt token or a token file
-    def authorize(self, token=None, timeout=5):
-        return self.backend.authorize(token, timeout)
+    # Do authorization by passing a token file
+    def authorize(self, tokenfile: str = None, timeout=5):
```

Comment:
This is maybe a good naming convention to gradually introduce also in other places - use `token` when we refer to the contents of a token, and `tokenfile` when the content is supposed to be a path